src/main/java/org/itsallcode/openfasttrace/core/LinkedSpecificationItem.java

```diff
@@ -83,6 +97,16 @@ public Location getLocation()
         return this.item.getLocation();
     }
 
+    /**
+     * Get the tags associated with this item.
+     * 
+     * @return list of tags
+     */
+    public List<String> getTags()
+    {
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)